Document the usage of `hardWrap` option for markdown rendering (#1951)
gohugoio/hugoDocs: content/en/getting-started/configuration-markup.md

Some settings explained:

hardWrap

> : By default, Goldmark will omit the newlines inside a paragraph. Turn this to true will let the newlines be rendered as `<br>`.

Reviewer: ```suggestion
: By default, Goldmark ignores newlines within a paragraph. Set to `true` to render newlines as `<br>` elements.
```
